tests/test_client.py

```diff
@@ -127,34 +150,34 @@ def test_read_multi_vars(self):
         # build up our requests
         data_items = (S7DataItem * 3)()
 
-        data_items[0].Area = ctypes.c_int32(S7AreaDB)
-        data_items[0].WordLen = ctypes.c_int32(WordLen.Byte.value)
-        data_items[0].Result = ctypes.c_int32(0)
-        data_items[0].DBNumber = ctypes.c_int32(db)
-        data_items[0].Start = ctypes.c_int32(0)
-        data_items[0].Amount = ctypes.c_int32(4)  # reading a REAL, 4 bytes
-
-        data_items[1].Area = ctypes.c_int32(S7AreaDB)
-        data_items[1].WordLen = ctypes.c_int32(WordLen.Byte.value)
-        data_items[1].Result = ctypes.c_int32(0)
-        data_items[1].DBNumber = ctypes.c_int32(db)
-        data_items[1].Start = ctypes.c_int32(4)
-        data_items[1].Amount = ctypes.c_int32(4)  # reading a REAL, 4 bytes
-
-        data_items[2].Area = ctypes.c_int32(S7AreaDB)
-        data_items[2].WordLen = ctypes.c_int32(WordLen.Byte.value)
-        data_items[2].Result = ctypes.c_int32(0)
-        data_items[2].DBNumber = ctypes.c_int32(db)
-        data_items[2].Start = ctypes.c_int32(8)
-        data_items[2].Amount = ctypes.c_int32(2)  # reading an INT, 2 bytes
+        data_items[0].Area = c_int32(Area.DB.value)
```

Comment:
Here and in other places, if you use `IntEnum`, you can get rid of `.value` part
![image](https://github.com/gijzelaerr/python-snap7/assets/52915342/84cbceb0-71eb-4be3-acd1-930d8f736af0)